stdmicro/hardware/RTC/ds3231/DS3231.cpp

```diff
@@ -133,9 +138,10 @@ void DS3231::enableInterrupt(void)
 
     // a = 10, b = 11, c = 12 = 1100
     // 0x1c = 00011100
-    mBuffer[1] |= 0x1c;
+    mBuffer[1] |= 0x04;
```

Comment:
Might consider defined constants for these bitflags to improve readability

Reply:
Good call, I think I am going to put this off for a separate PR and combine it with updating the GD7965 in the same way. This will happen, likely, before additional low-level hardware support is added :)